CHANGELOG.md

```diff
@@ -1,3 +1,55 @@
+<a name="v0.3.0"></a>
+# [v0.3.0](https://github.com/qri-io/dataset/compare/v0.2.0...v) (2021-05-04)
+
+This release of the `dataset` package includes one major change, adding a `Stats` component, as well as a few minor changes and a bunch of bug fixes that are listed below.
+
+## Stats component
+Introducing the `stats` component, a top-level component that caches stats calculated from the body and the structure. We calculate and store different kinds of stats based on the content of the given column or fields. The different types as of this release are numeric, boolean, and string. We've moved the `qri/stats` package into `dataset` under the name `dsstats`.
```

Comment:
It doesn't really cache them anywhere, it provides the mechanics to quickly generate the stats using probabilistic structures and is no longer bound to size/time limitations as the older implementation.

Though there is/was some caching in the works from the cloud side.